spec/features/seeds/adding_seeds_spec.rb

```diff
@@ -17,12 +17,23 @@
     within "form#new_seed" do
       fill_in "Quantity:", :with => 42
       fill_in "Plant before:", :with => "2014-06-15"
+      fill_in "Days until maturity, min:", :with => 999
+      fill_in "Days until maturity, max:", :with => 1999
+      select "certified organic", :from => "Organic?"
+      select "non-certified GMO-free", :from => "GMO?"
+      select "heirloom", :from => "Heirloom?"
       fill_in "Description", :with => "It's killer."
       select "internationally", :from => "Will trade:"
       click_button "Save"
     end
 
     expect(page).to have_content "Successfully added maize seed to your stash"
+    expect(page).to have_content "Quantity: 42"
+    expect(page).to have_content "Days until maturity: 999"
```

Comment:
Sorry, shouldn't have said "date range" but an integer range? It's only testing for "days to maturity 55" not "days to maturity 55-65", see? But the partial he wrote displays only 1 number when min or max is blank and displays "x-y" when they're both filled in.